core/src/it/scala/org/ensime/intg/ReverseLookupsSpec.scala

```diff
@@ -102,39 +104,41 @@ class ReverseLookupsSpec extends EnsimeSpec
     }
   }
 
-  it should "only search files loaded into pres. compiler when reverse lookups are turned off" in withEnsimeConfig { implicit config =>
-    withTestKit { implicit testKit =>
-      withProject { (project, asyncHelper) =>
-        try {
-          Properties.setProp("ensime.index.no.reverse.lookups", "true")
-          import testKit._
-          val sourceRoot = scalaMain(config)
-          val fooFile = sourceRoot / "org/example/Foo.scala"
-
-          project ! RefactorReq(1234, RenameRefactorDesc("notATestMethod", fooFile, 119, 119), interactive = false)
-          expectMsgPF() {
-            case response @ RefactorDiffEffect(1234, RefactorType.Rename, diff) =>
-              val relevantExpectedPartFoo =
-                s"""|@@ -9,3 +9,3 @@
-                    |   class Foo extends Bar {
-                    |-    def testMethod(i: Int, s: String) = {
-                    |+    def notATestMethod(i: Int, s: String) = {
-                    |       i + s.length
-                    |@@ -16,3 +16,3 @@
-                    |   println("Hello, " + foo.x)
-                    |-  println(foo.testMethod(7, "seven"))
-                    |+  println(foo.notATestMethod(7, "seven"))
-                    | 
+  it should "only search files loaded into pres. compiler when reverse lookups are turned off" in {
+    try {
+      Properties.setProp("ensime.index.no.reverse.lookups", "true")
```

Comment:
this is tricky because while this test is running, other tests in the same JVM will not do reverse lookups. I'm ok with this being untested.